vignettes/references.bib

```diff
@@ -11,4 +11,145 @@ @Article{Isaksen2023
+  type    = {Journal Article},
+  url     = {https://www.cambridge.org/core/article/swedish-national-diabetes-register-in-clinical-practice-and-evaluation-in-primary-health-care/F3A9B74FED1E7B36147FF63382939B9A},
+}
+
+@Article{Cooper2013,
+  author  = {Cooper, John G. and Thue, Geir and Claudi, Tor and Løvaas, Karianne and Carlsen, Siri and Sandberg, Sverre},
+  journal = {Norsk Epidemiologi},
+  title   = {The Norwegian Diabetes Register for Adults – an overview of the first years},
+  year    = {2013},
+  number  = {1},
+  volume  = {23},
+  doi     = {10.5324/nje.v23i1.1599},
+  type    = {Journal Article},
+  url     = {https://www.ntnu.no/ojs/index.php/norepid/article/view/1599},
+}
+
+
+
+@Article{Carstensen2011,
+  author  = {Carstensen, B. and Kristensen, J. K. and Marcussen, M. M. and Borch-Johnsen, K.},
+  journal = {Scand J Public Health},
+  title   = {The National Diabetes Register},
+  year    = {2011},
+  issn    = {1403-4948},
+  number  = {7 Suppl},
+  pages   = {58-61},
+  volume  = {39},
+  doi     = {10.1177/1403494811404278},
+  type    = {Journal Article},
+}
+
+@Article{Green2014,
+  author  = {Green, Anders and Sortsø, Camilla and Jensen, Peter Bjødstrup and Emneus, Martha},
+  journal = {Clinical epidemiology},
+  title   = {Validation of the Danish National Diabetes Register},
+  year    = {2014},
+  issn    = {1179-1349},
+  pages   = {5-15},
+  volume  = {7},
+  doi     = {10.2147/CLEP.S72768},
+  type    = {Journal Article},
+  url     = {https://www.ncbi.nlm.nih.gov/pubmed/25565889
+https://www.ncbi.nlm.nih.gov/pmc/PMC4274151/},
+}
+
+@Misc{DHDA2016,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2016},
+  url    = {https://www.esundhed.dk/-/media/Files/Dokumentation/Register-for-Udvalgte-Kroniske-Sygdomme-og-Svaere-Psykiske-Lidelser/29_Algoritmer_for_RUKS.ashx},
+}
+
+@Misc{DHDA2018,
+  author = {{Danish Health Data Authority}},
+  title  = {{Dokumentation af Laboratoriedatabasens Forskertabel – Version 1.3 [English: Documentation on Register of Laboratory Results for Research - version 1.3]}},
+  year   = {2018},
+  url    = {https://sundhedsdatastyrelsen.dk/-/media/sds/filer/registre-og-services/nationale-sundhedsregistre/doedsaarsager-og-biologisk-materiale/laboratoriedatabasen/dokumentation-af-labdatabasens-forskertabel.pdf},
+}
+
+@Misc{SDHDA2022lastaccessed7Feb2023,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2022},
+  url    = {https://www.esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser-RUKS-2022.ashx},
+}
```

Comment:
```suggestion
```

This isn't even used in the document, I must have accidentally added it from the other bib file.